src/mumble/ListenerLocalVolumeSlider.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "ListenerLocalVolumeSlider.h"
+#include "Channel.h"
+#include "ChannelListenerManager.h"
+#include "VolumeAdjustment.h"
+#include "Global.h"
+
+ListenerLocalVolumeSlider::ListenerLocalVolumeSlider(QWidget *parent) : VolumeSliderWidgetAction(parent) {
+}
+
+void ListenerLocalVolumeSlider::setListenedChannel(const Channel &channel) {
+	m_channel = &channel;
+
+	blockSliderSignals(false);
```

Comment:
I'm wondering why explicitly _unblocking_ the signals is required here :thinking:

Reply:
Me too. Do you want me to try without the blocking. Hehe

Reply:
I think the signal blocking was implemented to prevent signals being process before channel or user is set to the object. I do not know if this race condition is even possible

Reply:
> I think the signal blocking was implemented to prevent signals being process before channel or user is set to the object.

Yes, but I don't think this is what happens here. I think for that we'd need `blockSliderSignals(true)`, don't we? If not, then the function will need a different name, imo.

In any case though, I think it would be worth just trying not to do any blocking to see if that has any effect :eyes:

Reply:
``blockSliderSignals(true)`` is in the base class constructor

Reply:
Ah okay - then that explains, why we require unblocking here ^^

Anyway - the blocking in general still seems unnecessary at first glance :thinking:

Reply:
yes, I will investigate soon if we need the blocking

Reply:
Seems to work without the signal blocking. There are nullptr checks anyway so I suppose nothing can go wrong...